### PR TITLE
examples: plot tracks in binding lifetime example

### PR DESCRIPTION
**Pull request by aafkevandenberg**
I added a plot of the tracks, to make more clear what we are actually analyzing in the rest of the example.

### CODE REVIEW

File: docs/examples/binding_lifetime/binding_lifetime.rst [hunk 1]
```diff
@@ -91,6 +91,18 @@ Load the tracks into Pylake::
 
 Use the same approach as above to load the tracks from Lakeview, except that the part :func:`Kymo.crop_by_distance() <lumicks.pylake.kymo.Kymo.crop_by_distance>` has to be removed.
 
+The (loaded) tracks can be plotted on top of the original kymograph to visualize the result of the tracking::
+
+    plt.figure()
+    kymo1_selection.plot("g", aspect = 5, adjustment=lk.ColorAdjustment([0], [5]))
```
Comment (JoepVanlier) [inline docs/examples/binding_lifetime/binding_lifetime.rst:97]: Minor, but you don't need to make the adjustments lists if you specify only a single value.

```suggestion
    kymo1_selection.plot("g", aspect=5, adjustment=lk.ColorAdjustment(0, 5))
```

File: docs/examples/binding_lifetime/binding_lifetime.rst [hunk 2]
```diff
@@ -91,6 +91,18 @@ Load the tracks into Pylake::
...
+    tracks1.plot()
+
+.. image:: tracks1.png
```
Comment (JoepVanlier) [inline docs/examples/binding_lifetime/binding_lifetime.rst:100]: Minor, but if you save the image with `plt.savefig("tracks1.png", bbox_inches="tight")`, you'll get a figure with less whitespace. Might look slightly better in the rendered docs.

**Review by JoepVanlier — APPROVED** [review]
Looks good to me! Two small suggestions, but nothing blocking.

File: docs/examples/binding_lifetime/binding_lifetime.rst (hunk 2 above)
Comment (aafkevandenberg) [reply docs/examples/binding_lifetime/binding_lifetime.rst:100]: Indeed, that looks much better. I'll also update the two images with kymographs, as they have a lot of white space.

**Pull request merged**